src/inverseRobot.js

```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (invertedRobot.hasOwnProperty(robot[key])) {
+      return null;
+    }
+    invertedRobot[robot[key]] = key;
```

Comment:
Would be great to create a variable for `robot[key]` value in order not to duplicate yourself and make your code cleaner